data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/RequestContext.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © 2018 Knative Authors (knative-dev@googlegroups.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package dev.knative.eventing.kafka.broker.receiver;
+
+import io.vertx.core.http.HttpServerRequest;
+
+public class RequestContext {
```

Comment:
is this wrapper really a "generic" context? IMO sounds broader than it is, since we just calculate the latency as an additional thing

Reply:
Are you suggesting to call it `RequestWrapper` rather than `RequestContext`?

Reply:
I am not sure - but I am also fine w/ the `RequestContext` (I do find naming hard, tbh)